### DISCUSSION TITLE
replace/supplement "full screen" button with "open in new tab"

### DISCUSSION THREAD

**Discussion by petergaultney** [pythonspeed/filprofiler#295] [Ideas]
The fullscreen option works, but I have a very wide monitor and could put two of these side by side at a large width for comparison.

I'd prefer to have a button that lets me open the SVG directly, but that doesn't force my browser into full screen mode, which is much less flexible than just having an open browser tab.

I can and do open the SVG directly, but that's not an ideal user experience in my humble opinion. :)

**Comment by itamarst** [comment]
That's a good idea! Would  "open in new window" make more sense for this use, or do you specifically want "new tab"?

**Reply by itamarst** [reply]
Oh, I guess that's not a thing, the browser decides.

**Reply by itamarst** [reply]
I opened an issue, #298.

**Reply by itamarst** [reply]
This is now available in latest release, 2022.1.0, available from PyPI (Conda-Forge is having issues, so may be a bit of time before that works).
